### DISCUSSION TITLE
Can fees be added in the admin panel when editing an order?

### DISCUSSION THREAD

**Discussion by kiancross** [pretix/pretix#3782] [Q&A] [answered]
Is it currently possible to add a fee when editing the order in the admin panel?

e.g., if we have an order with multiple positions, and want to cancel one position and apply a cancellation fee.

**Comment by kiancross** [comment]
It appears that the answer is no. Is this intentional, or just a missing feature?

**Reply by kiancross** [reply]
Additionally, is there a reason why fee descriptions are displayed on the admin panel, but not the order details page for the user?

**Reply by raphaelm** [reply, marked as answer]
> It appears that the answer is no. Is this intentional, or just a missing feature?

It is just a missing feature.

> Additionally, is there a reason why fee descriptions are displayed on the admin panel, but not the order details page for the user?

I think that one is intentional, as fee descriptions usually do not add relevant information for the user, they are more intended to e.g. split fees for different payment methods in reporting.
